storage/entities/peer.js

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const filterType = require('../utils/filter_types');
+const { stringToByte } = require('../utils/inputSerializers');
+const BaseEntity = require('./base_entity');
+
+const defaultCreateValues = {};
+
+const readOnlyFields = [];
+
+/**
+ * Basic Peer
+ * @typedef {Object} Peer
+ * @property {number} id
+ * @property {inet} ip
```

Comment:
We save the ip as `inet` in the database but I reckon we should return it as string here.

Reply:
And if fact it does return as a string, good catch! Done.